default_settings.json

```diff
@@ -0,0 +1,5254 @@
+	"editor.suggestSelection": "first",
+
+	// Enables tab completions.
+	//  - on: Tab complete will insert the best matching suggestion when pressing tab.
+	//  - off: Disable tab completions.
+	//  - onlySnippets: Tab complete snippets when their prefix match. Works best when 'quickSuggestions' aren't enabled.
+	"editor.tabCompletion": "off",
+
+	// The number of spaces a tab is equal to. This setting is overridden based on the file contents when `editor.detectIndentation` is on.
+	"editor.tabSize": 4,
+
+	// Overrides editor syntax colors and font style from the currently selected color theme.
+	"editor.tokenColorCustomizations": {},
+
+	// Remove trailing auto inserted whitespace.
+	"editor.trimAutoWhitespace": true,
+
+	// Controls whether clicking on the empty content after a folded line will unfold the line.
+	"editor.unfoldOnClickAfterEndOfLine": false,
+
+	// Defines allowed characters that are not being highlighted.
+	"editor.unicodeHighlight.allowedCharacters": {},
+
+	// Unicode characters that are common in allowed locales are not being highlighted.
+	"editor.unicodeHighlight.allowedLocales": {
+		"_os": true,
+		"_vscode": true
+	},
+
+	// Controls whether characters are highlighted that can be confused with basic ASCII characters, except those that are common in the current user locale.
+	"editor.unicodeHighlight.ambiguousCharacters": true,
+
+	// Controls whether characters in comments should also be subject to unicode highlighting.
+	"editor.unicodeHighlight.includeComments": "inUntrustedWorkspace",
+
+	// Controls whether characters in strings should also be subject to unicode highlighting.
+	"editor.unicodeHighlight.includeStrings": true,
+
+	// Controls whether characters that just reserve space or have no width at all are highlighted.
+	"editor.unicodeHighlight.invisibleCharacters": true,
+
+	// Controls whether all non-basic ASCII characters are highlighted. Only characters between U+0020 and U+007E, tab, line-feed and carriage-return are considered basic ASCII.
+	"editor.unicodeHighlight.nonBasicASCII": "inUntrustedWorkspace",
+
+	// Remove unusual line terminators that might cause problems.
+	//  - auto: Unusual line terminators are automatically removed.
+	//  - off: Unusual line terminators are ignored.
+	//  - prompt: Unusual line terminators prompt to be removed.
+	"editor.unusualLineTerminators": "prompt",
+
+	// Inserting and deleting whitespace follows tab stops.
+	"editor.useTabStops": true,
+
+	// Controls whether completions should be computed based on words in the document.
+	"editor.wordBasedSuggestions": true,
+
+	// Controls from which documents word based completions are computed.
+	//  - currentDocument: Only suggest words from the active document.
+	//  - matchingDocuments: Suggest words from all open documents of the same language.
+	//  - allDocuments: Suggest words from all open documents.
+	"editor.wordBasedSuggestionsMode": "matchingDocuments",
```

Comment:
Perhaps this should be `allDocuments` to e.g. complete controller variables in views?